pybossa/api/__init__.py

```diff
@@ -376,6 +379,57 @@ def get_disqus_sso_api():
         return error.format_exception(e, target='DISQUS_SSO', action='GET')
 
 
+@jsonpify
+@csrf.exempt
+@blueprint.route('/project/<short_name>/chat', methods=['POST'])
+@ratelimit(limit=ratelimits.get('LIMIT'), per=ratelimits.get('PER'))
+def chat_notify(short_name):
+    """Email project owners upon a user initiating a chat session."""
+    if not current_user.is_authenticated:
+        return abort(401)
+
+    data = request.json
+    project = project_repo.get_by_shortname(short_name)
+    if not project:
+        return abort(400)
+
+    try:
+        data = request.json
+        subject = 'Chat session started for project ' + short_name + ' by ' + current_user.email_addr
+        success_body = (
+            'A user has started a chat session on a project that you an owner/co-owner for.\n\n'
+            '    Project Short Name: {short_name}\n'
+            '    User requesting assistance: {user}\n'
+            '    Message: {message}\n\n'
+            'Slack Url\n'
+            'https://gigwork-dev.slack.com\n'
```

Comment:
can we have a config parameter for this?

Reply:
Added via https://github.com/bloomberg/pybossa/pull/400/commits/5be401fc9581b2ccfcd416b6c907b8e7f4cca3e7